docs/diginstroom/sip/1.0/profiles/newspaper.md

```diff
@@ -0,0 +1,346 @@
+---
+layout:       default
+title:        Newspaper
+parent:       Profiles
+nav_order:    2
+nav_exclude:  false
+---
+# Profile: Newspaper 
+
+The newspaper profile supports the ingest of digitised newspaper content.
+It is an extension of the generic SIP specification and it shows how to deal with multiple media files (in formats such as TIFF, ALTO XML and PDF) and the relationships between them and their metadata.
+It also allows to use the [MODS metadata schema](https://www.loc.gov/standards/mods/) for descriptive metadata, which is the default for describing newspaper content.
+
+**Permalink:** <https://data.hetarchief.be/id/sip/1.0/newspaper>
+
+**Example Directory structure:**
+
+```plaintext
+root_directory
+│──manifest-md5.txt
+│──bagit.txt
+│
+└──data
+    │──mets.xml
+    │──metadata
+    |   |──descriptive
+    |   |  └──dc.xml
+    |   |  └──mods.xml
+    |   └──preservation
+    |       └──premis.xml
+    │
+    └──representations
+        │──representation_1
+        │    │──mets.xml
+        │    │──data
+        │    |   |──file_1.tiff
+        │    │   └──...
+        │    │
+        │    └──metadata
+        │        └──preservation
+        │            └──premis.xml
+        │
+        │──representation_2
+        │    │──mets.xml
+        │    │──data
+        │    |   |──file_1.xml
+        │    │   └──...
+        │    │
+        │    └──metadata
+        │       └──preservation
+        │          └──premis.xml
+        │
+        └──representation_3
+            │──mets.xml
+            │──data
+            |  └── file_1.pdf
+            │
+            └──metadata
+               └──preservation
+                  └── premis.xml
+```
+
+## Requirements
+
+### General
+
+- A newspaper SIP MUST contain exactly one newspaper edition.
+- The newspaper edition MUST be digitised per page, i.e. each TIFF and/or ALTO XML file contained in their respective representation directories MUST represent exactly one page.
+  - <a id="pdf"></a>An exception to this requirement MAY be made with regards to the PDF file: if a PDF file is present, it SHOULD contain the contents of the entire newspaper edition (i.e. all pages are present in one single PDF file).
```

Comment:
However, there's a Python package PyPDF2 with which you can count the number of pages of a PDF file; this would enable an automatic check? You could check whether the number of pages in the PDF equals the number of TIFFs in the filesec of the TIFF representation METS?

Reply:
Fair enough, but are we sure about this assumption on `#tiffs = #pdf pages`?

Reply:
For the time being I'll leave it as is. If/when we accept newspaper digitised per article, we can revise the requirement.